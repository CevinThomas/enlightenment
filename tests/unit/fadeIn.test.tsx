import 'react-native';
import React from 'react';
import FadeIn from '../../src/components/fadeIn';

import renderer from 'react-test-renderer';

test('FadeIn Component Renders', () => {
    const tree = renderer.create(
        <FadeIn/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
