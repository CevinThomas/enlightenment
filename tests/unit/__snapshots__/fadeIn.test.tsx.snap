// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FadeIn Component Renders 1`] = `
<View
  style={
    Object {
      "opacity": 0,
    }
  }
/>
`;
